bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java

```diff
@@ -58,77 +64,60 @@ class SecurityCapability extends RestCapability<SecurityApi> {
+                        modules.values().stream().filter(module -> childId.equals(module.getBridge()))
+                                .forEach(bridgedModule -> childHandler.setNewData(bridgedModule));
+                    });
         });
     }
 
     @Override
     protected void updateHomeStatus(HomeStatus homeStatus) {
         NAObjectMap<HomeStatusPerson> persons = homeStatus.getPersons();
         NAObjectMap<HomeStatusModule> modules = homeStatus.getModules();
-        handler.getActiveChildren().forEach(childHandler -> {
+        handler.getActiveChildren(FeatureArea.SECURITY).forEach(childHandler -> {
             String childId = childHandler.getId();
             persons.getOpt(childId).ifPresentOrElse(personData -> childHandler.setNewData(personData), () -> {
-                modules.getOpt(childId).ifPresentOrElse(childData -> {
+                modules.getOpt(childId).ifPresent(childData -> {
                     childHandler.setNewData(childData);
                     modules.values().stream().filter(module -> childId.equals(module.getBridge()))
-                            .forEach(bridgedModule -> {
-                                childHandler.setNewData(bridgedModule);
-                            });
-
-                }, () -> {
-                    // This module is not present in the homestatus data, so it is considered as unreachable
-                    HomeStatusModule module = new HomeStatusModule();
-                    module.setReachable(false);
-                    childHandler.setNewData(module);
+                            .forEach(bridgedModule -> childHandler.setNewData(bridgedModule));
                 });
             });
         });
     }
 
     @Override
     protected void updateHomeEvent(HomeEvent homeEvent) {
-        String personId = homeEvent.getPersonId();
-        if (personId != null) {
-            handler.getActiveChildren().stream().filter(handler -> personId.equals(handler.getId())).findFirst()
-                    .ifPresent(handler -> {
-                        homeEvent.setIgnoredForThingUpdate(true);
-                        handler.setNewData(homeEvent);
-                    });
+        addEventIfKnownObject(homeEvent, homeEvent.getPersonId());
+        addEventIfKnownObject(homeEvent, homeEvent.getCameraId());
+    }
+
+    private void addEventIfKnownObject(HomeEvent homeEvent, @Nullable String objectId) {
+        if (objectId == null) {
+            return;
         }
-        String cameraId = homeEvent.getCameraId();
-        handler.getActiveChildren().stream().filter(handler -> cameraId.equals(handler.getId())).findFirst()
-                .ifPresent(handler -> {
-                    homeEvent.setIgnoredForThingUpdate(true);
-                    handler.setNewData(homeEvent);
-                });
+        handler.getActiveChildren().stream().filter(child -> child.getId().equals(objectId))
```

Comment:
No need to pass FeatureArea.SECURITY here ?

Reply:
Yes, even better.